events_backend/app/urls.py

```diff
@@ -0,0 +1,64 @@
+"""events_backend URL Configuration
+
+The `urlpatterns` list routes URLs to views. For more information please see:
+    https://docs.djangoproject.com/en/2.0/topics/http/urls/
+Examples:
+Function views
+    1. Add an import:  from my_app import views
+    2. Add a URL to urlpatterns:  path('', views.home, name='home')
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.conf.urls import include, url
+from django.contrib.auth import views as auth_views
+from django.views.generic import TemplateView
+from rest_framework.authtoken import views as authviews
+from . import views
+
+dateRegex = "[0-9]{4}[0-1][0-9][0-3][0-9]T[0-9]{6}"
+spoof = "Clny9jLW4lf2gAvOZ27fJYnPJNyEPqKiMfmkAkFgXzksjoTEkeL9o5M4lHNbyrUCsVnglEv89pTcs1o787xt17KF6HZaHuqDMEjNPfpDdJBeB8nHbFLqJuqGTsEqT5NAai7UGJxgSPcszLTmpMT6PhRY7VATmEcbbqvf3McjsvfZ0Um9UHUVzCwpnj67n6rSbiy4kZm8"
```

Comment:
still dunno what this is, probably should remove?

Reply:
yeah just going to remove it. It is technically used in one of the urls but yeah I don't see what it does too.